types/markdown-it/lib/rules_inline/state_inline.d.ts

```diff
@@ -0,0 +1,48 @@
+import * as MarkdownIt from "..";
+import * as State from "../rules_core/state_core";
+import * as Token from "../token";
+
+export = StateInline;
+
+declare class StateInline extends State {
+    /**
+     * Stores `{ start: end }` pairs. Useful for backtrack
+     * optimization of pairs parse (emphasis, strikes).
+     */
+    cache: { [start: number]: number };
+
+    /** Emphasis-like delimiters */
+    delimiters: MarkdownIt.Delimiter[];
+
+    pending: string;
+    pendingLevel: number;
+
+    /** Index of the first character of this token. */
+    pos: number;
+
+    /** Index of the last character that can be used (for example the one before the end of this line). */
+    posMax: number;
+
+    /**
+     * Push new token to "stream".
+     * If pending text exists, flush it as text token.
+     */
+    push(type: string, tag: string, nesting: number): Token;
+
+    /** Flush pending text */
+    pushPending(): Token;
+
+    /**
+     * Scan a sequence of emphasis-like markers and determine whether
+     * it can start an emphasis sequence or end an emphasis sequence.
+     * @param start - position to scan from (it should point to a valid marker)
+     * @param canSplitWord - determine if these markers can be found inside a word
+     */
+    scanDelims(start: number, canSplitWord: boolean): {
+        can_open: boolean,
+        can_close: boolean,
+        length: number
+    };
+}
+
+declare module StateInline {}
```

Comment:
This is an inappropriate hack. [Please read more here from the DefinitelyTyped README](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

Additionally, test files should ideally use the `import Foo = require("foo")` style syntax.

Reply:
all done!